lib/krikri.rb

```diff
@@ -12,4 +12,5 @@ module Krikri
   autoload :OaiDcParser,    'krikri/parsers/oai_dc_parser'
   autoload :JsonParser,     'krikri/parsers/json_parser'
   autoload :ModsParser,     'krikri/parsers/mods_parser'
+  autoload :QdcParser,      'krikri/parsers/qdc_parser'
 end
```

Comment:
I took a look at our use of `autoload` in other modules; currently, only [`krikri/ldp`](https://github.com/dpla/KriKri/blob/develop/lib/krikri/ldp.rb) and [`krikri/enrichments`](https://github.com/dpla/KriKri/blob/develop/lib/krikri/enrichments.rb) are using them. We don't have a `Krikri::Parsers` module, however.